fe/src/main/java/org/apache/doris/master/ReportHandler.java

```diff
@@ -388,21 +389,26 @@ private static void sync(Map<Long, TTablet> backendTablets, ListMultimap<Long, L
                 List<Long> tabletIds = tabletSyncMap.get(dbId);
                 LOG.info("before sync tablets in db[{}]. report num: {}. backend[{}]",
                          dbId, tabletIds.size(), backendId);
-
-                for (Long tabletId : tabletIds) {
-                    long tableId = invertedIndex.getTableId(tabletId);
+                List<TabletMeta> tabletMetaList = invertedIndex.getTabletMetaList(tabletIds);
```

Comment:
Have you tested its performance? There may be millions of tablet in `tabletIds`. I am not sure how long it will take to process that much of tablets.

Reply:
cost for getting a list of size for 1000000 from map  is about 40 ~ 50ms, here we only preocess tablets in one be and which need to sync, so i think  1000000 for testing is large enough

Reply:
OK